scripts/utils/index.d.ts

```diff
@@ -0,0 +1,4 @@
+export { findConfig } from './find-config';
```

Comment:
this pattern provides proper typings in TS scope, without need of manually duplicating definitions like we do in `scripts/monorepo/index.d.ts`